packages/app/documents/service.ts

```diff
@@ -95,11 +95,12 @@ export async function createDocument(
         }
 
         //* This logic (and associated TODO) is ported from Meditor.js, saveDocument. Minimal modifications were made.
-        const rootState = { source: INIT_STATE, target: initialState }
+        const modDate = new Date().toISOString()
```

Comment:
Small nit, took me a bit to figure out what modDate was, may be better spelled out, like `modifiedOnDate`